common/autoware_auto_perception_rviz_plugin/include/object_detection/object_polygon_detail.hpp

```diff
@@ -80,7 +80,7 @@ AUTOWARE_AUTO_PERCEPTION_RVIZ_PLUGIN_PUBLIC visualization_msgs::msg::Marker::Sha
 get_shape_marker_ptr(
   const autoware_auto_perception_msgs::msg::Shape & shape_msg,
   const geometry_msgs::msg::Point & centroid, const geometry_msgs::msg::Quaternion & orientation,
-  const std_msgs::msg::ColorRGBA & color_rgba);
+  const std_msgs::msg::ColorRGBA & color_rgba, const double line_width = 0.03);
```

Comment:
```suggestion
  const std_msgs::msg::ColorRGBA & color_rgba, const double & line_width);
```

I'm not sure but it is natual for me.